packages/react-composites/src/composites/CallComposite/adapter/CallAdapter.ts

```diff
@@ -232,6 +240,29 @@ export interface CallAdapterCallManagement {
    * @public
    */
   removeParticipant(userId: string): Promise<void>;
+  /* @conditional-compile-remove(PSTN-calls) */
+  /**
+   * Adds a participant to the call by dialing them in
+   * @param participant - Phone number or Communication userId for participant being added
+   * @param options - Call options containing ACS number for PSTN calls
+   * @beta
+   */
+  addParticipant(
+    participant: PhoneNumberIdentifier | CommunicationUserIdentifier | UnknownIdentifier | MicrosoftTeamsUserIdentifier,
```

Comment:
Instead of creating a union of this type everywhere. Can we use the `CommunicationIdentifier` exported by communication-common?